frontend/src/components/PMMainPage.js

```diff
@@ -233,7 +230,6 @@ export class PMMainPage extends Component {
           <ModalHeader>Add New Field Partner</ModalHeader>
           <ModalBody>
             <form onSubmit={this.handleNewFP}>
-              <p>Organization Name:</p>
```

Comment:
why is this removed?

Reply:
The placeholder value is Enter Organization name here so I felt as if that label was redundant

Reply:
The placeholder does disappear after you type something tho, so redundancy might be good here